Epsilon/Export/Exporters/ConsoleModuleExporter.cs

```diff
@@ -13,29 +12,25 @@ public ConsoleModuleExporter(ILogger<ConsoleModuleExporter> logger)
     {
         _logger = logger;
     }
-
+    
     public IEnumerable<string> Formats { get; } = new[] { "console", "logs" };
 
-    public async Task Export(IAsyncEnumerable<ModuleOutcomeResultCollection> data, string format)
+    public async Task Export(IEnumerable<Module> data, string format)
```

Comment:
Instead of IEnumerable<Module> we can maybe use a standard data model for this so that we can easily change it without having to update the interface.

Reply:
I think this is already the case? If not, correct me and could you also by any chance then send me an example?
![image](https://user-images.githubusercontent.com/76601644/221655346-0087de30-719f-4cb3-a6ef-61cbfb6223e6.png)